docs/dev_guide/ceph_s3_tests_guide.md

```diff
@@ -0,0 +1,138 @@
+# Ceph S3 Tests Guide
+## Introduction
+* Ceph S3 tests are unofficial AWS S3 compatibility tests (for more information see: [ceph/s3-test](https://github.com/ceph/s3-tests) repository on GitHub) that we use in our CI.
+* This guide contains 3 parts:
+     1) General Settings For Ceph S3 Tests
+     2) Run a Single Ceph S3 Test (also contains debugging instructions)
+     3) Run All Ceph S3 Tests
+* This guide is written for developers that wants to run those tests on a noobaa system on minikube.
+
+## General Settings For Ceph S3 Tests
+
+### 1) Before Building The Images (Noobaa-Core)
+We will use minikube to run the tests. It is recommended to build all images on the minikube docker daemon. Configure your docker client to use minikube's docker run:
+```bash
+eval $(minikube docker-env)
+```
+
+
+### 2) Build Operator (Noobaa-Operator)
+We want to build the CLI and the operator image.
+It's easiest to just build `all`, run:
+```bash
+. ./devenv.sh
+make all
```

Comment:
Maybe it's obvious, but maybe it's worth explcitly mentioning that the commands that build the images (this make and the make in core) has to be run from the terminal that executed devenv.sh.

Reply:
You're right. I'm assuming that the user runs the commands from separate tabs (I added a clarification).